package.json

```diff
@@ -31,7 +31,9 @@
     "watch2": "tsc -watch -p ./",
     "package": "vsce package",
     "publish": "vsce publish",
-    "package-ext": "webpack --mode production --config ./build/extension.webpack.config.js"
+    "package-ext": "webpack --mode production --config ./build/extension.webpack.config.js",
+    "sbt": "sbt universal:packageBin",
+    "build": "yarn sbt && yarn install && yarn compile && yarn package"
```

Comment:
Cool!

Just a question about yarn, is this standard to have a yarn script run other yarn scripts like this? Thinking how sbt works, you just specify command dependencies. So you could specify that `yarn package` depends on `yarn compile`, which depends on `yarn install`. That way you can tell people to just run `yarn package` and if they haven't run the previous commands, they would get run because they are dependent commands.

But maybe yarn doesn't work like that, and this `yarn build` is the standard way to do this kind of thing?

Reply:
From what I saw it seems the only way to do is the way I have implemented. Yarn doesn't allow you to specify other scripts a script depends on so that they will run prior.

I believe the only way it functions this way is if you have something like `build` and `prebuild` then run `yarn run build` I think it will automatically run prebuild first but not completely sure on that.